pyproject.toml

```diff
@@ -1,4 +1,126 @@
-[tool.black]
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "yasa"
+description = "YASA: Analysis of polysomnography recordings."
+readme = "README.rst"
+license = {text = "BSD (3-clause)"}
+authors = [
+    {name = "Raphael Vallat", email = "raphaelvallat9@gmail.com"},
+    {name = "Remington Mallett", email = "mallett.remy@gmail.com"},
+]
+maintainers = [
+    {name = "Raphael Vallat", email = "raphaelvallat9@gmail.com"},
+    {name = "Remington Mallett", email = "mallett.remy@gmail.com"},
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dynamic = ["version"]
+requires-python = ">=3.8"
```

Comment:
bump to `>=3.9`, consistent with the classifiers in this .toml file and in the python_tests workflow. (of course bc 3.8 is at EOL)